V7 documentation update (#256)
neo4j/docs-graphql: modules/ROOT/pages/types/relationships.adoc

    title: String!
    released: Int!
    actors: [Person!]! @relationship(type: "ACTED_IN", direction: IN)
    directors: [Person!]! @relationship(type: "DIRECTED", direction: IN)
}
----

Note that, in this query:

* A `Person` can _act in_ or _direct_ multiple movies, and a `Movie` can have multiple actors. 

>   While it is rare for a `Movie` to have more than one director, Neo4j GraphQL Library requires all relationships to be modeled as "many" relationships (using Lists).

Reviewer: ```suggestion
  While it is rare for a `Movie` to have more than one director, the Neo4j GraphQL Library requires all relationships to be modeled as "many" relationships (using lists).
```

> * Even conceptually one-to-one relationships like a movie having a single director must be represented as arrays (e.g., `directors: [Person!]!`) because Neo4j cannot reliably enforce 1:1 cardinality between nodes.

Reviewer: ```suggestion
* Even conceptually one-to-one relationships, such as a movie having a single director, must be represented as arrays (e.g., `directors: [Person!]!`) because Neo4j cannot reliably enforce one-to-one cardinality between nodes.
```

* To figure out whether the `direction` argument of the `@relationship` directive should be `IN` or `OUT`, visualize your relationships like in the diagram above, then model out the direction of the arrows.
* The `@relationship` directive is a reference to Neo4j relationships, whereas in the schema, the phrase `edge(s)` is used to be consistent with the general API language used by Relay.

> === Neo4j GraphQL Cardinality Limitation

Reviewer: ```suggestion
=== Neo4j GraphQL cardinality limitation
```

> It's important to understand that Neo4j cannot reliably enforce 1:1 cardinality between nodes in a graph database. Therefore, all relationships in Neo4j GraphQL are modeled as "many" relationships (using Lists) even when conceptually they represent one-to-one relationships.

Reviewer: ```suggestion
It's important to understand that Neo4j cannot reliably enforce one-to-one cardinality between nodes in a graph database.
Therefore, all relationships in Neo4j GraphQL are modeled as "many" relationships (using lists) even if they represent one-to-one relationships conceptually.
```

> As alternative you can define your relationship with the `@cypher` directive, however some of the functionality provided by the Neo4j GraphQL Library will not be available.

Reviewer: ```suggestion
As an alternative you can define your relationship with the `@cypher` directive, however some of the functionality provided by the Neo4j GraphQL Library will not be available.
```

what kind of functionality would then be missing?
